tests/lib/outputs/outputs_test.py

```diff
@@ -337,9 +336,7 @@ def test_send_to_s3_bucket(self):
         client.create_bucket(Bucket=bucket_name)
         # Create mock csv output file
         fixtures_dir = "fixtures"
-        output_directory = (
-            f"{pathlib.Path().absolute()}/tests/lib/outputs/{fixtures_dir}"
-        )
```

Comment:
it would be interesting to test both cases to check if we are covering the old and the new paths. What do you think?

Reply:
Sure, done!